### PR TITLE
refactor: Improve warnings and error messages to users about sharing

### PR DESCRIPTION
**Pull request by krynble**

### CODE REVIEW

File: packages/cli/src/credentials/credentials.controller.ts [hunk 1]
```diff
@@ -160,7 +160,7 @@ credentialsController.patch(
 				userId: req.user.id,
 			});
 			throw new ResponseHelper.ResponseError(
-				`Credential with ID "${credentialId}" could not be found to be updated.`,
+				'Credential not found to be updated. You can only update credentials owned by you',
```
Comment (alexgrozav) [inline packages/cli/src/credentials/credentials.controller.ts:163]: Better wording: `Credential to be updated not found`

File: packages/cli/src/credentials/credentials.controller.ts [hunk 2]
```diff
@@ -218,7 +218,7 @@ credentialsController.delete(
 				userId: req.user.id,
 			});
 			throw new ResponseHelper.ResponseError(
-				`Credential with ID "${credentialId}" could not be found to be deleted.`,
+				'Credential not found to be deleted. You can only removed credentials owned by you',
```
Comment (alexgrozav) [inline packages/cli/src/credentials/credentials.controller.ts:221]: Better wording: `Credential to be deleted not found`

File: packages/cli/src/workflows/workflows.controller.ee.ts [hunk 3]
```diff
@@ -140,7 +144,7 @@ EEWorkflowController.post(
 			EEWorkflows.validateCredentialPermissionsToUser(newWorkflow, allCredentials);
 		} catch (error) {
 			throw new ResponseHelper.ResponseError(
-				'The workflow contains credentials that you do not have access to',
+				'The workflow you trying to save contains credentials not shared with you',
```
Comment (alexgrozav) [inline packages/cli/src/workflows/workflows.controller.ee.ts:147]: Better wording: `The workflow you are trying to save contains credentials that are not shared with you`

**Review by alexgrozav — APPROVED** [review]

**Comment by janober** [comment]
Got released with `n8n@0.204.0`

**Pull request merged**